### DISCUSSION TITLE
Organizational settings need to be improved

### DISCUSSION THREAD

**Discussion by AdminHcat** [community/community#27992] [Other Feature Feedback, Questions, & Ideas]
The position of the change visibility button in the organization page should be improved
If there are too many members in the organization, it takes a long time to search yourself

Even some organizations can't search for themselves at all so there is no way to complete the setting
eg. EpicGames

#18268 #12534
> https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-membership-in-organizations/publicizing-or-hiding-organization-membership

**Comment by jfullstackdev** [comment]
first click 

![image](https://user-images.githubusercontent.com/47092464/182314245-163be910-8600-481f-9c02-dc62e2e9462f.png)

second click

![image](https://user-images.githubusercontent.com/47092464/182314293-ffab0d6b-b96d-46c5-a8ff-aab9fb39b60b.png)

third click 

![image](https://user-images.githubusercontent.com/47092464/182314393-6f18604a-cb64-48a3-aec9-ba6772600a90.png)

you can even bookmark the URL 

`https://github.com/orgs/[org-name]/people`

**Reply by AdminHcat** [reply]
GitHub limits the number of members displayed, which makes it impossible for organizations with more than 50000 people to set these

**Reply by jfullstackdev** [reply]
yes, that's for sure, why, searching is not working ?
